source/features/deep-reblame.tsx

```diff
@@ -75,9 +75,9 @@ async function redirectToBlameCommit(event: delegate.Event<MouseEvent, HTMLAncho
 		blameUrl.branch = await getPullRequestBlameCommit(prCommit, prNumber, blameUrl.filePath);
 		blameUrl.hash = 'L' + select('.js-line-number', blameHunk)!.textContent!;
 		location.href = String(blameUrl);
-	} catch (error) {
+	} catch (error: unknown) {
```

Comment:
Judging by the alert, we expect an `Error`

Reply:
But the rule only allows `any` or `unknown`

Reply:
Indeed. Sounds like a useless rule. Let's disable it instead

Reply:
What is the point of the rule? Sindre just enabled in that last update.

Reply:
`any` lets you use variable however you want to use them while `unknown` forces you to type them before use. That's my understanding.

`unknown` forces us to check the type of the error before using it, but if we're just going to use assertions everywhere this isn't nearly a helpful.

We can leave it enabled